Modules/Loadable/Markups/MRML/vtkCurveGenerator.h

```diff
@@ -185,6 +185,9 @@ class VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkCurveGenerator : public vtkPolyDa
   /// Get the output sampled points
   vtkPoints* GetOutputPoints();
 
+  static void SortByIndex(vtkPoints*, vtkDoubleArray*);
```

Comment:
Could you add the name of the parameters: 

```
static void SortByIndex(vtkPoints* inputControlPoints, vtkDoubleArray* sortedControlPoints);
```

For SortByIndex ... it seems it is done more then just sorting:

https://github.com/Slicer/Slicer/blob/eb2b1f41ac7b3dc655a5e05b569ad70ffe4ea7fb/Modules/Loadable/Markups/MRML/vtkCurveGenerator.cxx#L782

Reply:
Now this is part of the public API, adding doxygen would be nice too.

Reply:
OK, doxygen added!

Reply:
Thanks